### ISSUE TITLE
PSR12.Files.FileHeader: false positive when there is no header

### ISSUE THREAD

**Issue by jrfnl** [PHPCSStandards/PHP_CodeSniffer#21]
Repost from https://github.com/squizlabs/PHP_CodeSniffer/issues/2997:

> Given the following code run against the PSR12 standard:
> ```php
> <?php
> 
> /*
>  * Not a file docblock.
>  */
> 
> /**
>  * Docblock for an include statement.
>  */
> include 'foo.php';
> ```
> 
> I get the following error:
> ```
> --------------------------------------------------------------------------------------
> FOUND 1 ERROR AFFECTING 1 LINE
> --------------------------------------------------------------------------------------
>  9 | ERROR | [x] Header blocks must be separated by a single blank line
>    |       |     (PSR12.Files.FileHeader.SpacingAfterBlock)
> ```
> 
> ... while I expected none.

---

See the original thread for additional information.
